includes/AppLeave.php

```diff
@@ -163,6 +168,7 @@
 	
 	$destDB = str_replace('.','$',$appr);
 	$destDB = str_replace(' ','_',$destDB);
+	
```

Comment:
__[code_sniffer]__
* Spaces must be used for alignment; tabs are not allowed

<!-- Comment by [SideCI](https://sideci.com) -->